compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -29,8 +29,8 @@ impl MutVisitor for Marker {
 enum Frame<'a> {
     Delimited {
         tts: &'a [mbe::TokenTree],
-        delim_token: token::DelimToken,
         idx: usize,
+        delim_token: token::DelimToken,
```

Comment:
In the final diff this field mysteriously moved.
Probably better to merge commits adding and removing `Option` to this structure.

Reply:
It's simpler than that: in the third commit, while changing this type, I chose to reorder the fields to better the `Sequence` variant :)

Reply:
I will add a note to the commit message about this change.